Entities/MovableObject.cpp

```diff
@@ -539,20 +546,25 @@ int MovableObject::LoadScript(const std::string &scriptPath, bool loadAsEnabledS
 
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-int MovableObject::ReloadScripts() {
-    if (m_LoadedScripts.empty()) {
+int MovableObject::ReloadScripts(bool alsoReloadPresetcripts) {
```

Comment:
Got a typo here. `alsoReloadPresetcripts` -> `alsoReloadPresetScripts`.
Same in the header.